extensions/amp-access-laterpay/0.1/amp-access-laterpay.js

```diff
@@ -30,18 +35,120 @@ export class LaterpayVendor {
   constructor(accessService) {
     /** @private @const */
     this.accessService_ = accessService;
-    // TODO: implement
+    this.win_ = this.accessService_.win;
+    this.laterpayConfig_ = this.accessService_.adapter_.getConfig();
```

Comment:
We'll need to expose this as a public method on `AccessService`, e.g. `getConfig()`. This is because all private vars are obfuscated.